extension.ts

```diff
@@ -169,6 +170,9 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
     );
     registerDebugConfigurationProvider(ctx);
 
+    let imageGrouping: number | undefined = ext.context.globalState.get<ImageGrouping>(groupImagesByKey);
```

Comment:
This falls under what I would consider temporary (local) user settings. Don't see much sense in sharing this, for example, and you're likely to switch between different settings in a session. Doesn't seem like scc should be invoked on that. Discoverability is not an issue since it's listed right in the images node title and the menus. Also the exact groupings may still be a bit experimental.

I could see perhaps having a default grouping in settings, but don't want to deal with that now for similar reasons.

cc @fiveisprime